src/index.html

```diff
@@ -1,17 +1,155 @@
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <section>
+      <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+        method="post">
+        <fieldset class="fieldset">
+          <legend>Personal information</legend>
+          <label for="sorname" class="control-label">
+           Sorname:
+            <input
+            type="text"
+            name="sorname"
+            id="sorname"
+            >
+          </label>
+          <label for="name" class="control-label">
+            Name:
+            <input
+            type="text"
+            name="name"
+            id="name"
+            >
+          </label>
+          <label for="age" class="control-label">
+            How old are You?
+            <input
+            type="number"
+            name="age"
+            id="age"
+            min="0"
+            max="100"
+            value="12"
+            >
+          </label>
+          <label for="birthDate" class="control-label">
+            Full date of birth:
+            <input
+            type="date"
+            name="birthDate"
+            id="birthDate"
+            >
+          </label>
+          <label for="acceptAgreement" class="control-label">
+            I accept the term of the agreement
+            <input
+            type="checkbox"
+            name="acceptAgreement"
+            id="acceptAgreement"
+            value="no"
+            required
+            >
+          </label>
+        </fieldset>
+        <fieldset class="fieldset">
+          <legend>Registration</legend>
+          <label for="email" class="control-label">
+            E-mail:
+            <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+            >
+          </label>
+          <label for="password" class="control-label">
+            Password:
+            <input
+            type="password"
+            name="password"
+            id="password"
+            minlength="6"
+            maxlength="20"
+            required
+            >
+          </label>
+        </fieldset>
+        <fieldset class="fieldset">
+          <legend>An interesting fact about you!</legend>
+          <label class="control-label">
+            Do you love cats?
+            <input
+            type="radio"
+            name="cats"
+            value="yes"
+            > Yes 
+            <input
+            type="radio"
+            name="cats"
+            value="no"
+            > No 
+          </label>
```

Comment:
create separate labels for these inputs. now when I click on `Do you love cats?` it selects the first radio input (`Yes`)